File: ppmtool/src/main/java/com/lige110/springreact/ppmtool/repositories/ProjectTaskRepository.java
package com.lige110.springreact.ppmtool.repositories;

import com.lige110.springreact.ppmtool.domain.ProjectTask;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ProjectTaskRepository extends CrudRepository<ProjectTask,Long> {
}

File: ppmtool/src/main/kotlin/com/lige110/ppmtool/repositories/ProjectTaskRepository.kt
package com.lige110.ppmtool.repositories

import com.lige110.ppmtool.domain.ProjectTask
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface ProjectTaskRepository : CrudRepository<ProjectTask, Long> {
    fun findByProjectIdentifierOrderByPriority(id: String): List<ProjectTask>
    fun findByProjectSequence(projectSequence: String): ProjectTask?
}
